packages/components/src/internal/components/domainproperties/samples/SampleTypeDesigner.tsx

```diff
@@ -608,6 +613,7 @@ class SampleTypeDesignerImpl extends React.PureComponent<Props & InjectedBaseDom
                             ? getDomainPanelStatus(1, currentPanelIndex, visitedPanels, firstState)
                             : 'COMPLETE'
                     }
+                    showStudyPropertyTypes={_showLinkToStudy}
```

Comment:
using the `_showLinkToStudy` here for `showStudyPropertyTypes` ties these two properties together for sample types. Do we want that? 

Since the only place right now that looks to set `showLinkToStudy={false}` is LKB, this would mean that the study property types would also not show for LKB sample types as well. This is a bit of a "regression" since LKB currently shows the Particpant/Subject option in the LKB app for sample types field types. 

With the existing `allowTimepointProperties` property on the domain design object, would the check for `hasModule("study")` here be enough. I think if the study module isn't present we surely shouldn't be showing these study related property types, but I'm hoping that is enough of a check. Thoughts?

Reply:
The Participant/Subject option still appears in LKB.  Such a confusion of configurations!

Let's talk about the `allowTimepointProperties` and `hasModule` checks.

Reply:
As discussed, I'm going to leave this as is for now.  We do, currently, want to tie these two properties together and we don't always want to show the link to study even if the study module is present since that's the Biologics case but do want to check for the study module presence here rather than relying on the user to do that (since that's the non-premium LKSM case).